--- vendor/spryker/setup/src/Spryker/Zed/Setup/Business/Model/Cronjobs.php	2019-01-23 12:51:13.067451000 +0000
+++ vendor/spryker/setup/src/Spryker/Zed/Setup/Business/Model/Cronjobs.php.new	2019-01-23 12:54:48.748754000 +0000
@@ -368,11 +368,12 @@
   <triggers class='vector'>$schedule</triggers>
   <concurrentBuild>false</concurrentBuild>
   <builders>
-    <hudson.tasks.Shell>";
+    <org.jvnet.hudson.plugins.SSHBuilder plugin='ssh@2.6.1'>
+        <siteName>jenkins@appHost:222</siteName>";
 
         $xml .= $this->getCommand($command, $store);
         $xml .= "
-    </hudson.tasks.Shell>
+    </org.jvnet.hudson.plugins.SSHBuilder>
   </builders>\n"
             . $this->getPublisherString($job) . "\n
   <buildWrappers/>
@@ -459,8 +460,6 @@
         $commandTemplate = '<command>%s
 export APPLICATION_ENV=%s
 export APPLICATION_STORE=%s
-cd %s
-. %s
 %s</command>';
 
         $cronjobsConfigPath = $this->config->getCronjobsConfigFilePath();
@@ -470,11 +469,10 @@
         $destination = APPLICATION_ROOT_DIR;
 
         if ($environment->isNotDevelopment()) {
-            $checkDeployFolderExistsBashCommand = '[ -f ' . APPLICATION_ROOT_DIR . '/deploy/vars ]';
-            $sourceBashCommand = '. ' . APPLICATION_ROOT_DIR . '/deploy/vars';
+            $checkDeployFolderExistsBashCommand = '[ -f ' . '/versions/vars ]';
+            $sourceBashCommand = '. ' . '/versions/vars';
 
             $customBashCommand = $checkDeployFolderExistsBashCommand . ' ' . '&amp;&amp;' . ' ' . $sourceBashCommand;
-            $destination = '$destination_release_dir';
         }
 
         return sprintf(
@@ -482,8 +480,6 @@
             $customBashCommand,
             $environment->getEnvironment(),
             $store,
-            $destination,
-            $cronjobsConfigPath,
             $command
         );
     }
