--- src/Pyz/Zed/Mail/MailDependencyProvider.php 2018-12-27 10:23:47.550483085 +0000
+++ src/Pyz/Zed/Mail/MailDependencyProvider.php 2018-12-27 10:23:56.918412986 +0000
@@ -20,6 +20,10 @@
 use Spryker\Zed\Newsletter\Communication\Plugin\Mail\NewsletterUnsubscribedMailTypePlugin;
 use Spryker\Zed\Oms\Communication\Plugin\Mail\OrderConfirmationMailTypePlugin;
 use Spryker\Zed\Oms\Communication\Plugin\Mail\OrderShippedMailTypePlugin;
+use Spryker\Zed\Mail\Dependency\Mailer\MailToMailerBridge;
+use Swift_Mailer;
+use Swift_Message;
+use Swift_SmtpTransport;

 class MailDependencyProvider extends SprykerMailDependencyProvider
 {
@@ -28,6 +32,29 @@
      *
      * @return \Spryker\Zed\Kernel\Container
      */
+
+     protected function addMailer(Container $container)
+     {
+         $container[static::MAILER] = function () {
+             $message = new Swift_Message();
+             $transport = new Swift_SmtpTransport(
+                 $this->getConfig()->getSmtpHost(),
+                 $this->getConfig()->getSmtpPort()
+             );
+              $transport->setHost(getenv('SMTP_HOST'));
+              $transport->setPort(getenv('SMTP_PORT'));
+              $transport->setUsername(getenv('SMTP_USER'));
+              $transport->setPassword(getenv('SMTP_PASS'));
+              $transport->setEncryption("ssl");
+
+             $mailer = new Swift_Mailer($transport);
+
+             return new MailToMailerBridge($message, $mailer);
+         };
+
+         return $container;
+     }
+
     public function provideBusinessLayerDependencies(Container $container)
     {
         $container = parent::provideBusinessLayerDependencies($container);

